tendrl/alerting/handlers/cluster/brick_utilization_handler.py

```diff
@@ -10,6 +10,48 @@ class BrickUtilizationHandler(AlertHandler):
     def __init__(self):
         AlertHandler.__init__(self)
 
+    def format_alert_message(self):
```

Comment:
Implementation of this function looks more or less similar in all the cases. Only thing changing is name/description of the plugin instance. Can we try to make it generic and have implemented in base class itself with messages appending plugin instance name/description?